servlet/src/main/java/io/undertow/servlet/spec/ServletOutputStreamImpl.java

```diff
@@ -332,31 +352,35 @@ public void write(ByteBuffer[] buffers) throws IOException {
                     Buffers.copy(buffer, buffers, 0, buffers.length);
                 } else {
                     final ByteBuffer[] bufs = new ByteBuffer[buffers.length + 1];
-                    buffer.flip();
```

Comment:
Buffer flip should not be removed. I'll update the PR since we will be doing the release today.